exercises/scripts/Setup-LocalDBInstance.ps1

```diff
@@ -1,8 +1,14 @@
 #Requires -RunAsAdministrator
 
-$instanceName = "microservices-workshop"
+$instanceName = "particular-workshop"
 
 sqllocaldb create $instanceName
 sqllocaldb share $instanceName $instanceName
 sqllocaldb start $instanceName
 sqllocaldb info $instanceName
+
+$serverName = "(localdb)\" + $instanceName
+$userprofilepath = "$ENV:UserProfile\particular-workshop-databases"
```

Comment:
nitpick: This variable does not represent the path user profile. It represents the path to the databases. I would rename to `$databasesPath`.